### DISCUSSION TITLE
How to select non clustered resource types to replicate

### DISCUSSION THREAD

**Discussion by d4rkhunt33r** [Azure/fleet#773] [Q&A]
Hi

I am looking at the documentation and I cannot find how  to replicate non clustered resource types. 

According to the docs, ClusterResourcePlacement selects only cluster-scoped resources.

That means I cannot select for example all Kubernetes services from the cluster to replicate to another cluster ?

**Comment by zhiying-lin** [comment]
@d4rkhunt33r when the CRP selects the namespace, it will "// If a namespace is selected, ALL the resources under the namespace are placed to the target clusters."

https://github.com/Azure/fleet/blob/main/apis/placement/v1/clusterresourceplacement_types.go#L44

If you want to select a particular resource in a namespace instead of ALL, it's in our roadmap.
